### ISSUE TITLE
/f claim , Commands Error spam

### ISSUE THREAD

**Issue by Jnxd1** [MassiveCraft/Factions#1005]
Hello there guys i have a non stop errors incoming when players use mostly the /f claim ,commands.I have just saw this yestoday and i don't know if there are more going on but i will try to track em down and give you back the reports on them.I guess they are using normally the commands and they are in use cuz no one of them didnt tell me nthn for something not working, its just the console spam  as fare as i know.
I em using 2.8.2 and running it on spigot 1.8.8 (just updated few days ago im about 4 vers. behind).
Here are some of the error that i did track up, sorry i think 1 may be a double copy.
http://pastebin.com/vWkpb3m8
http://pastebin.com/4PxAE5Bz
http://pastebin.com/tf9VSQuG
http://pastebin.com/waDrUKkB
http://pastebin.com/qVtXeNnP
Thanks so much guys, gj to the team.
Cheers!

**Comment by ulumulu1510** [comment]
in some of these pastebins, I see this line:
17.01 08:19:54 [Multicraft] Skipped 23 lines due to rate limit (30/s)
Please disable this rate limit because those 20-30 lines could be really helpful for us.

The first pastebin lets me believe that your MassiveCore and Factions versions aren't synchronized, could that be? java.lang.NoClassDefFoundError: com/massivecraft/factions/entity/FactionColls happens when a class is no present, which should not be the case at all.

Please also tell me some more about your servers setup, whether you use bungeecord or anything alike.

**Comment by Jnxd1** [comment]
Okey i will have to connect my host so they can  disable the rate limit.
And sorry im kinda new to you plugin MassiveCore and Factions i tough that the Massive Core was just the base for it and didn't need any updates.
And yes i run a bungee cord, and on this server is only the Massive Core + Factions.
Please let me know if you need more info that i could provide like plugins and other.

**Comment by ulumulu1510** [comment]
You need to make sure that the versions of MassiveCore and Factions are the same aka. synchronized. I have not have had much experience with running Factions in a bungeecord setup, but I can imagine that as long as the versions of MassiveCore and Factions are synchronized on each of the servers, it will not pose a problem. 

Please tell me which exact versions you are using (after you updated) please. And tell me if the issues persist after synchronizing the versions. Also, keep in mind that it is important to keep them snychronized, just for the coming updates.

**Comment by Jnxd1** [comment]
Sorry for the delay i had some things to attend to, so here are the full error's logs that you needed.
http://pastebin.com/8AKjrs5c
http://pastebin.com/asghAn4r
http://pastebin.com/ZvwFqnzL
I hope this will work for you guys finding out what is wrong, and im running on Massive Core 2.8.2
Tell me if you need something else and i will provide 
Thanks !

**Comment by magnusulf** [comment]
The cause for this is Gringotts. Gringotts is severely outdated and tries to use an extremely old version of Factions.

**Comment by Jnxd1** [comment]
Ohhh i see...
Is there any fix..., well i know updated version of Gringotts but can it be done something else?
All of my server is based over Gringotts, it is essential for it.
Or if i can't find a way out of it, can i continue with this error spaming?I mean is it so severe or i can do ok?
Thanks so much for the fast response and rlly guys good work on the Faction!

**Comment by ulumulu1510** [comment]
With your current errors, no transaction from or to a faction account (or faction bank, however you want to call it) will be successful. Simply put, there is no proper solution without updating gringots.

You can of course live with the error. If you still want your player to pay for land claiming and get awarded for land purge, you can't do anything about it. If you are willing to sacrifice the economic features of factions, you can turn the econ enabled option off in your config.

Again, this is not the proper solution. But it might work.

**Comment by Jnxd1** [comment]
Hello again, thanks for the fast responses guys rlly!
I wanted to ask you, i had some kind of a server crash which grew in a problem that i could not start the server (just if you want to see it here is the paste bin http://pastebin.com/zjMbJFuG ).I did a fast search on the web and i found a post at spigot that was saying to try and delete the "usercache.json" file.Before i delete it i checked it and it was empty so i did so and deleted it (i also save it at my pc).
Now after i joined the faction server i had my "nick" restarted (which i think its from the usercache.json) , but i also was not in the faction that i have created and i get this message in the console, http://pastebin.com/akUt2bxr . Btw i em running a offline server and when i created my faction i was in online mode.Is there chance that there was some kinda UDDI mess up there?And can i delete the file?
And one last thing, i tried to delete my faction that i have created but i get an internal error http://pastebin.com/rSh3h3Ft .
I have just downloaded the Factions 2.8.4 zip from Spigot.
Thank you again guys!
